packages/ui/app/Header.tsx

```diff
@@ -25,6 +26,7 @@ const LINK = {
   [AppType.Kashi]: '/internal',
   [AppType.Analytics]: '/analytics',
   [AppType.Pool]: '/pool',
+  [AppType.Partner]: '/partner',
```

Comment:
I dont think we should add it to the menu, instead add it to the footer. Its not really a product hence it shouldnt be in the menu

Reply:
it is not in the "active/public" navbar, had to add it here so that it shows up when you're on the page

Reply:
Is it not showing now when you're on the swapper and click the menu?

Reply:
it's not no

Reply:
Alright cool, lgtm then!